README.md

```diff
@@ -46,6 +47,15 @@ Useful links:
 
 Generation of artificial neuronal trees based on the topology of biological cells and their statistical properties.
 
+## Axon-Synthesis
+<img alt="Axon-Synthesis Banner" src="https://github.com/BlueBrain/axon-synthesis/raw/main/docs/source/BBP-Axon-Synthesis.jpg" width="600"/>
+
+Useful links:
+[GitHub repo](https://github.com/BlueBrain/axon-synthesis),
+[Documentation](https://axon-synthesis.readthedocs.io/).
+
+**Long-range axon synthesis in brain atlases.**
+
```

Comment:
may be add a bit of text after the title describing your repo

Reply:
Done